MonkeyWrench.Database/DB.cs

```diff
@@ -499,7 +500,7 @@ public DBLane CloneLane (int lane_id, string new_name, bool copy_files)
 
 					foreach (DBHostLaneView hostlanemaster in master.GetHosts (this)) {
 						DBHostLane clone = new DBHostLane ();
-						clone.enabled = false;
+						clone.enabled = hostlanemaster.enabled;
```

Comment:
I think as long as the cloned lane itself is disabled, nothing will be done for the hostlanes, even if they are enabled, since the revisions won't be added to the DB. There is sufficient time for the user to edit the fields before the lane gets enabled for the first time and starts populating the DB with revisions.

This is one less manual edit we need to do when cloning the lane.